build-linux/build.sh

```diff
@@ -26,26 +26,26 @@ if [ "${CLASSICAL}" == "1" ]; then
 
   export PATH="${GODOT_SDK_LINUX_X86_64}/bin:${BASE_PATH}"
 
-  $SCONS platform=linuxbsd $OPTIONS tools=yes target=release_debug
+  $SCONS platform=linuxbsd arch=x86_64 $OPTIONS tools=yes target=release_debug
   mkdir -p /root/out/x86_64/tools
   cp -rvp bin/* /root/out/x86_64/tools
   rm -rf bin
 
-  $SCONS platform=linuxbsd $OPTIONS tools=no target=release_debug
-  $SCONS platform=linuxbsd $OPTIONS tools=no target=release
+  $SCONS platform=linuxbsd arch=x86_64 $OPTIONS tools=no target=release_debug
+  $SCONS platform=linuxbsd arch=x86_64 $OPTIONS tools=no target=release
   mkdir -p /root/out/x86_64/templates
   cp -rvp bin/* /root/out/x86_64/templates
   rm -rf bin
 
   export PATH="${GODOT_SDK_LINUX_X86}/bin:${BASE_PATH}"
 
-  $SCONS platform=linuxbsd $OPTIONS tools=yes target=release_debug bits=32
+  $SCONS platform=linuxbsd arch=x86_32 $OPTIONS tools=yes target=release_debug bits=32
   mkdir -p /root/out/x86_32/tools
   cp -rvp bin/* /root/out/x86_32/tools
   rm -rf bin
 
-  $SCONS platform=linuxbsd $OPTIONS tools=no target=release_debug bits=32
-  $SCONS platform=linuxbsd $OPTIONS tools=no target=release bits=32
+  $SCONS platform=linuxbsd arch=x86_32 $OPTIONS tools=no target=release_debug bits=32
+  $SCONS platform=linuxbsd arch=x86_32 $OPTIONS tools=no target=release bits=32
```

Comment:
You can also remove `bits=32`.

Reply:
As far as I know, `bits=32` is still necessary if we want compatibility with the current master.

Reply:
Ah I didn't think that we currently use `arch` if it's defined over `bits` for the binary name. I thought you were trying to implement compatibility with https://github.com/godotengine/godot/pull/55778.

So yeah this might work as is.

Reply:
The current master has this in SConstruct:

```py
    if env["arch"] != "":
        suffix += "." + env["arch"]
    elif env["bits"] == "32":
        suffix += ".32"
    elif env["bits"] == "64":
        suffix += ".64"
```